src/scripts/link-to-wallet.ts

```diff
@@ -2,7 +2,7 @@ import { MetaMaskInpageProvider } from "@metamask/providers"
 import detectEthereumProvider from "@metamask/detect-provider"
 import { BigNumberish, ethers } from "ethers"
 import { Web3Provider } from "@ethersproject/providers"
-import LinkToken from "@chainlink/contracts/abi/v0.4/LinkToken.json"
```

Comment:
Hi @snehaagni 

the complied ABIs are not part of the npm package anymore?

Reply:
Next beta version will include them. Keeping this comment open until we restore this import.

Reply:
Resolved